### PR TITLE
Add notRestoredReasons property

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

The `PerformanceNavigationTiming.notRestoredReasons` property (see [Back/forward cache notRestoredReasons API](https://developer.chrome.com/docs/web-platform/bfcache-notrestoredreasons/)) is finally going to be launched in Chrome 117, or possibly 118 (See [ChromeStatus page](https://chromestatus.com/feature/5684908759449600)). This PR adds BCD for it.

### CODE REVIEW

**Review by Elchi3 — COMMENTED** [review]
Would it be better to wait until there is a spec url?

**Comment by chrisdavidmills** [comment]
> Would it be better to wait until there is a spec url?

Yes, I think so; I've been told this is imminent, but, converting to draft for now.

File: api/PerformanceNavigationTiming.json [hunk 1]
```diff
@@ -300,6 +300,39 @@
           }
         }
       },
+      "notRestoredReasons": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceNavigationTiming/notRestoredReasons",
+          "support": {
+            "chrome": {
+              "version_added": "116"
```
Comment (tunetheweb) [inline api/PerformanceNavigationTiming.json:308]: Turns out this did not ship in 116 after all. It's likely to be 117 but might slip to 118. Keep an eye on https://chromestatus.com/feature/5684908759449600 (which at the time of writing incorrectly states it did ship in 116).

Comment (chrisdavidmills) [reply api/PerformanceNavigationTiming.json]: Thx @tunetheweb. I've updated the version to 117 for now, and will look out for other changes.

File: api/PerformanceNavigationTiming.json [hunk 2]
```diff
@@ -526,4 +559,4 @@
       }
     }
   }
-}
+}
```
Comment (queengooborg) [inline api/PerformanceNavigationTiming.json:562]:
```suggestion
}

```

File: api/PerformanceNavigationTiming.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/PerformanceNavigationTiming.json]: @tunetheweb The ChromeStatus entry said it shipped in 117. Did it? ;-)

Comment (tunetheweb) [reply api/PerformanceNavigationTiming.json]: Nope! Got them to move forward to 118 but wouldn't surprise me if that slips again. There's some discussion on the [HTML standard repo PR](https://github.com/whatwg/html/pull/9360) so looking to get that all ironed out before enabling by default.

**Comment by queengooborg** [comment]
Chrome 119 is stable and it appears that this has still not shipped.  I think that we should close this PR and wait until a) the feature is standard track, and b) there is confirmed support in a stable browser release.

Note that when these two clauses are true, the mdn-bcd-collector project will let us automatically add it to BCD.

**Pull request closed without merging**